src/util/check-ref-resource.ts

```diff
@@ -5,17 +5,19 @@
 
 import { cli } from '../cli';
 import { Command } from '../cli-command';
-import { ContextType } from '../context-type';
-import { PipelineTaskRunData } from '../tekton';
+import { TknPipeline } from '../tekton';
 import { TektonNode } from '../tree-view/tekton-node';
 import * as vscode from 'vscode';
 
-export async function getTaskRunResourceList(): Promise<PipelineTaskRunData[]> {
-  const result = await cli.execute(Command.listTaskRun());
-  if (result.error) {
-    // ignore
-  }
-  let data: PipelineTaskRunData[] = [];
+const tektonResource = {
+  task: 'Task',
+  clustertask: 'ClusterTask'
+}
+
+
+export async function getPipelineList(): Promise<TknPipeline[]> {
```

Comment:
My suggestion is just to make sure you don't have this same code somewhere else. Maybe you needed to find the pipeline list in another place in your code? You can think about unifying them. Maybe it's not true, you know the code better than me, just an idea looking at it